core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -187,6 +188,14 @@ default void waitTopicDeletion(String topic) throws InterruptedException {
         waitForTopic(topic, 0);
     }
 
+    void waitForMeatdataSync(String topic, int partition, int isrSize) throws InterruptedException;
```

Comment:
this is almost equal to `waitForTopic`, right?

Reply:
Yes, I find `KRaft` can trace cluster log by controller but there is no way to check `cluster log` for `ZK`.
`ZK` only support partition level log so I am so hesitate to add this method.
We can discuss offline tonight!

Reply:
> this is almost equal to `waitForTopic`, right?

Hi @chia7712 , I removed `waitForMetadataSync`, PTAL.